core/org.testeditor.core/src/main/java/org/testeditor/core/headless/HeadlessTestRunnerApplication.java

```diff
@@ -240,18 +239,8 @@ public TestStructure getTestStructureToExecute(String[] args)
 	 */
 	public void initializeSystemConfiguration() throws BackingStoreException, IOException {
 		TestEditorConfigurationService testEditorConfigService = getService(TestEditorConfigurationService.class);
-		String waittime = System.getProperty(TestEditorGlobalConstans.DEFINE_WAITS_AFTER_TEST_STEP);
-		String browserPath = System.getProperty(TestEditorGlobalConstans.PATH_BROWSER);
-		testEditorConfigService.exportGlobalVariablesToSystemProperties();
+		testEditorConfigService.exportGlobalVariablesToSystemProperties(false);
```

Comment:
I think the readability would improve by refactoring the variable name to `configService`, same for `TestEditorConfigurationServiceImplIntTest`